person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindly ensure that `Getters and Setters for @rentals are present`. Add the `:rentals` to attr_accessor 👍.